dex/tests/close-markets/test-ledger/faucet-keypair.json

```diff
@@ -0,0 +1 @@
+[51,25,32,112,253,186,120,178,132,56,151,168,91,160,122,154,117,160,171,251,54,237,31,2,28,59,107,255,123,185,170,178,78,126,17,2,234,141,213,6,223,198,48,183,22,255,168,118,101,198,44,192,215,120,202,139,189,94,202,62,129,164,227,33]
```

Comment:
needs deletion?

Reply:
also the ledger.lock file below

Reply:
For sure. Not clear how the test-ledger file snuck in